e2e/wdio/core/tests/vertical-navigation.e2e-spec.ts

```diff
@@ -11,7 +11,7 @@ import {
     isElementClickable
 } from '../../driver/wdio';
 
-describe('Vertical navigation component tests', function () {
+describe('Vertical navigation component tests', function (): void {
```

Comment:
I guess, for consistency it would be better to just make it an arrow function instead. It won't complain about return type
```suggestion
describe('Vertical navigation component tests', () => {
```

Reply:
Done